cyber.latex

```diff
@@ -0,0 +1,264 @@
+\documentclass[$if(fontsize)$$fontsize$,$endif$$if(lang)$$babel-lang$,$endif$$if(papersize)$$papersize$paper,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{$documentclass$}
```

Comment:
Why are you using a template?  Could you accomplish the same things using the `-V` options in pandoc